lib/src/position.dart

```diff
@@ -535,9 +600,10 @@ abstract class Position<T extends Position<T>> {
     if (isLegal(move)) {
       final san = _makeSanWithoutSuffix(move);
       final newPos = playUnchecked(move);
-      final suffixed = newPos.outcome?.winner != null
```

Comment:
I also think that here we should override `playToSan` in `ChessSharp` to improve readability. Since this method is quite simple, code duplication is not really a problem imo.